### PR TITLE
Remove tapENABLE_TESTING.

### PR DESCRIPTION
**Pull request by scottschurr**
The `tapENABLE_TESTING` flag has been replaced by Feature.  This pull request removes the last vestiges of the flag.

Reviewers: @ximinez @vinniefalco

### CODE REVIEW

File: src/ripple/rpc/impl/TransactionSign.cpp [hunk 1]
```diff
@@ -696,8 +696,7 @@ Json::Value transactionSign (
     Role role,
     std::chrono::seconds validatedLedgerAge,
     Application& app,
-    std::shared_ptr<ReadView const> ledger,
-    ApplyFlags flags)
+    std::shared_ptr<ReadView const> ledger)
```
Comment (vinniefalco) [inline src/ripple/rpc/impl/TransactionSign.cpp]: `std::shared_ptr<ReadView const> const& ledger`?

**Comment by vinniefalco** [comment]
Hope coverage goes up
:+1:

**Comment by scottschurr** [comment]
@vinniefalco Regarding coverage, I hope so too, but we'll see.  This pull request removes tested lines of code (but I don't think it adds any untested lines).  So the total coverage may decrease.  I still think it's the right thing to do.

File: src/ripple/rpc/impl/TransactionSign.cpp (hunk 1 above)
Comment (scottschurr) [inline src/ripple/rpc/impl/TransactionSign.cpp]: Good eyes @vinniefalco.  Thanks for catching that.  Will fix.

**Comment by ximinez** [comment]
Looks good. :+1: 

The new "missed" lines in the coverage report seem to be the same kind of timing fluke affecting #1489.
Develop: https://codecov.io/github/ripple/rippled/src/ripple/app/main/LoadManager.cpp?ref=d7935192dd9db58463a6ac38a0906d0b9d78ecc8#l-182
This PR: https://codecov.io/github/ripple/rippled/src/ripple/app/main/LoadManager.cpp?ref=f21e83b4572175f0b25dc7a8370c9e32bee0b365#l-182

**Comment by ximinez** [comment]
Merged 0fca91c6c1f748f8c065b77075c6f812b3556be3

**Pull request closed without merging**